gcs-fetcher/cmd/gcs-uploader/main.go

```diff
@@ -54,7 +54,7 @@ func main() {
 	}
 	bucket, object, generation, err := common.ParseBucketObject(*location)
 	if err != nil {
-		log.Fatalln("parsing location from %q: %v", *location, err)
+		log.Fatalf("parsing location from %q: %v", *location, err)
```

Comment:
\n at the end

Reply:
`log` methods already append a newline at the end. I'm not sure why both `log.Print` and `log.Println` exist...

Reply:
cool

Reply:
lgtm